metals-docs/src/main/scala/docs/GenericModifier.scala

```diff
@@ -15,17 +15,20 @@ class GenericModifier extends StringModifier {
        |The Metals server places logs and other files in the `.metals` directory. The
        |Bloop compile server places logs and compilation artifacts in the `.bloop`
        |directory. Bloop plugin that generates Bloop configuration is added in the 
```

Comment:
```suggestion
       |directory. The Bloop plugin that generates Bloop configuration is added in the 
```
I know you didn't change this line, but just spotted this.

Reply:
Thanks for noticing that!